big_tests/tests/cets_disco_SUITE.erl

```diff
@@ -1,7 +1,7 @@
 -module(cets_disco_SUITE).
 -compile([export_all, nowarn_export_all]).
 
--import(distributed_helper, [mim/0, rpc/4]).
+-import(distributed_helper, [mim/0, mim2/0, rpc/4]).
```

Comment:
`mim2` seems to be unused. Btw - this suite looks like small tests to me, because we are using only one node, and there are no network connections (XMPP/RPC) used.

Reply:
yeah, that is why I like hank :)

Reply:
though I bring mim2 back, because now we use it to do tests in the meck version of the test.